deploy/sdn-ovn-migration/100-network-live-migration.PrometheusRule.yaml

```
@@ -0,0 +1,53 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    prometheus: sre-network-live-migration
+    role: alert-rules
+  name: sre-network-live-migration
+  namespace: openshift-monitoring
+spec:
+  groups:
+  - name: sre-network-live-migration
+    rules:
+      - expr:
+            (
+              max(
+                max_over_time(
+                  timestamp(
+                    openshift_network_operator_live_migration_condition{type="NetworkTypeMigrationInProgress"} == 1
+                  )[5d:]
+                )
+              )
+              -
+              min(
+                min_over_time(
+                  timestamp(
+                    openshift_network_operator_live_migration_condition{type="NetworkTypeMigrationInProgress"} == 1
+                  )[5d:]
+                )
+              )
+            )
+            and on()
+            openshift_network_operator_live_migration_condition{type="NetworkTypeMigrationInProgress"} == 1
+        record: cluster:usage:network_live_migration_duration
+      - expr: openshift_network_operator_live_migration_blocked
+        record: cluster:usage:network_live_migration_blocked
```

```suggestion
      - expr: openshift_network_operator_live_migration_blocked
        record: cluster:usage:network_live_migration_blocked
      - expr: openshift_network_operator_live_migration_condition
        record: cluster:usage:network_live_migration_condition
```
Looking great! Just need one more addition. See my [earlier comment](https://github.com/openshift/managed-cluster-config/pull/2258#pullrequestreview-2410519297) for details

---

Done :)